standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/AbstractJDBCConnectorProvider.java

```diff
@@ -176,7 +199,7 @@ protected Connection getConnection() {
     Table table = null;
     try {
       // rs = fetchTableMetadata(tableName);
```

Comment:
nit: I know this is old code. But can you please delete the commented line as well?